### PR TITLE
OGM-1116 Be consistent with Search/Validator in the way modules are built

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/OGM-1116

### CODE REVIEW

File: modules/pom.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
     <artifactId>hibernate-ogm-modules</artifactId>
     <packaging>pom</packaging>
 
-    <name>Hibernate OGM JBoss Modules</name>
-    <description>Create JBoss Modules for Hibernate OGM</description>
+    <name>Hibernate OGM Wildfly Module</name>
+    <description>Hibernate OGM modules creation to use in JBoss AS / Wildfly</description>
```
Comment (gunnarmorling) [inline modules/pom.xml]: It's WildFly only.

**Comment by gunnarmorling** [comment]
Looks good. The nested structure was there because at some point we had two ZIPs: one for WF and one for EAP, but the latter got outdated and I don't think it will come back. So +1 for this change.

**Comment by gsmet** [comment]
Fixed the wording and rebased.

**Comment by gunnarmorling** [comment]
Merged; thanks, @gsmet!

**Comment by gunnarmorling** [comment]
Ah, can you add an entry to the migration notes about the changed artifact id and classifier? Thanks!

**Pull request merged**